### PR TITLE
Adding Navigation component

### PR DESCRIPTION
**Pull request by SiantyTown**

### CODE REVIEW

File: FrontEnd/src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+class Navigation extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			pages: [
+				{
+					icon: 'book',
+					text: 'Library',
+					ref: '/index.html',
```
Comment (ZoltanT-RD) [inline FrontEnd/src/components/Navigation.js:19]: These will have to be changed later on, but ok for now

File: FrontEnd/src/colors.scss [hunk 2]
```diff
@@ -2,6 +2,8 @@
     --gold-primary: #BB8F52;
     --gold-accent: rgba(201, 170, 136, 30);
 
+    --blue-primary: #D0DBEA;
```
Comment (ZoltanT-RD) [inline FrontEnd/src/colors.scss]: hmm, you've added this, but never used it?

File: FrontEnd/src/components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+///section css
+import componentCSS from './Navigation.scss';
+
+class Navigation extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			pages: [
```
Comment (ZoltanT-RD) [inline FrontEnd/src/components/Navigation.js:15]: this probs. should be called "NaviationElement" or something along these lines, and use its own class

File: FrontEnd/src/components/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+				{
+					icon: 'face',
+					text: 'Admin',
+					ref: '/admin.html',
+					class: 'valign-wrapper',
+				},
+			],
+		};
+
+		switch (this.props.isActive) {
```
Comment (ZoltanT-RD) [inline FrontEnd/src/components/Navigation.js:37]: not sure this is the best way of doing this, also coz' this won't be able to switch between active elements just now, at least without changing the prop above

File: FrontEnd/src/colors.scss (hunk 2 above)
Comment (SiantyTown) [reply FrontEnd/src/colors.scss]: left over from testing i think

**Review by ZoltanT-RD — APPROVED** [review]

**Pull request merged**